### DISCUSSION TITLE
Disney+ 歪踢化

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#422] [General]
RT 以後應該不會推出免費但要一直看廣告的Disney+吧
應該不會吧...

https://technews.tw/2022/05/18/disney-plus-with-ads-will-keep-the-breaks-down-to-four-minutes-per-hour/

**Comment by brian431** [comment]
> RT 以後應該不會推出免費但要一直看廣告的Disney+吧
> 
> 應該不會吧...
> 
> 
> 
> 
> 
> https://technews.tw/2022/05/18/disney-plus-with-ads-will-keep-the-breaks-down-to-four-minutes-per-hour/
出了也未必是一件壞事，有些人就是寧願浪費自己的時間也不願花錢。這樣對於公司來說可以拓展自己的客源，收入增加後自然也能提升影片的品質及數量。

**Comment by HsuChen88** [comment]
雖然我不是數學家，但感覺他先開放免費加入廣告後，會再出Premium重回訂閱制，賺了廣告的錢，也賺了訂閱會員的錢。

**Comment by lukiea** [comment]
看來是沒梗了 只能說在現在的串流平台下，DISNEY+最大的優勢就是情懷，將一些過往我這些電視小孩的回憶，重回到視野中，感到欣慰的同時，也不禁讓我深深懷疑，那接下來呢，它們還能搬出啥，雖然在曼威在一連串的影集攻勢下看起來有道爭取串流平台龍頭的氣勢，但不得不說，接下來就沒啥東西可以看了，大家還是乖乖回去各自習慣的串流平台，希望接下來可以看到，他們真的在這個串流平台推出自己引以為傲的系列

**Comment by larry40305** [comment]
現在youtube也在走下坡，希望有更多影音平台供消費者使用
